nzedb/Releases.php

```diff
@@ -199,7 +200,50 @@ public function getBrowseRange($cat, $start, $num, $orderBy, $maxAge = -1, $excl
 	{
 		$orderBy = $this->getBrowseOrder($orderBy);
 
-		$qry = sprintf(
+        if ($this->hideuploadednzb) {
```

Comment:
Duplicating the entire query is bad.  Just add an additional %s to the existing query and do the substitution in the correct place like this:
```
 +                FROM
+                (
+                    SELECT r.*, g.name AS group_name
+                    FROM releases r
+                    LEFT JOIN groups g ON g.id = r.group_id
+                    WHERE r.nzbstatus = %d
+                    %s
+                    AND r.passwordstatus %s
+                    %s %s %s %s
+                    ORDER BY %s %s %s
+                ) r

+                ORDER BY %7\$s %8\$s",
+                NZB::NZB_ADDED,
+                (isset($this->hideuploadednzb) ? 'AND r.name NOT LIKE "%%.nzb\" yEnc' : ''),
+                $this->showPasswords,
+                $this->categorySQL($cat),
+                ($maxAge > 0 ? (" AND postdate > NOW() - INTERVAL " . $maxAge . ' DAY ') : ''),